.github/workflows/test-integrations-miscellaneous.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.5","3.8","3.11","3.12"]
+        python-version: ["3.8","3.11","3.12"]
```

Comment:
Should we be testing 3.6 and/or 3.7 here?

Reply:
We should -- the yaml files will be updated as soon as I add these to tox.ini